rafay/resource_import_cluster.go

```diff
@@ -93,10 +100,24 @@ func resourceImportCluster() *schema.Resource {
 			"kubernetes_provider": {
 				Type:     schema.TypeString,
 				Optional: true,
+				ValidateDiagFunc: func(i interface{}, p cty.Path) diag.Diagnostics {
```

Comment:
Is the validate function called only if the value is filled?

Reply:
yes it's called if resource field is specified in resource.tf file